homeassistant/components/tado/__init__.py

```diff
@@ -425,3 +435,12 @@ def set_temperature_offset(self, device_id, offset):
             self.tado.set_temp_offset(device_id, offset)
         except RequestException as exc:
             _LOGGER.error("Could not set temperature offset: %s", exc)
+
+    def set_meter_reading(self, reading: int) -> dict[str, str] | None:
+        """Send meter reading to Tado."""
+        try:
+            return self.tado.set_eiq_meter_readings(reading=reading)
+        except RequestException as exc:
+            _LOGGER.error("Could not set meater reading: %s", exc)
```

Comment:
```suggestion

```

Reply:
this is obsolete now, right?

Reply:
No, `set_meter_reading` is the registered method in the service.

Reply:
ah right sorry